internal/flex/flex.go

```diff
@@ -43,7 +43,7 @@ func ExpandStringList(configured []interface{}) []*string {
 
 // Takes list of pointers to strings. Expand to an array
 // of raw strings and returns a []interface{}
-// to keep compatibility w/ schema.NewSetschema.NewSet
+// to keep compatibility w/ schema.NewSet
```

Comment:
This may be unintentional.

Reply:
Looked like an accidental double `ctrl`+`v` to me, especially when compared to `FlattenInt64List` below https://github.com/cloudposse/terraform-provider-awsutils/blob/c441d598810c6118dfa6be479c951ccbadc60c4c/internal/flex/flex.go#L104